Core/Core/People/PeopleListViewController.swift

```diff
@@ -262,6 +263,7 @@ class PeopleListCell: UITableViewCell {
         avatarView.url = user?.avatarURL
         let nameText = user.flatMap { User.displayName($0.name, pronouns: $0.pronouns) }
         nameLabel.setText(nameText, style: .textCellTitle)
+        nameLabel.accessibilityIdentifier = "\(self.accessibilityIdentifier!).name-label"
```

Comment:
force unwrap is always risky, I'd suggest using self.accessibilityIdentifier ?? "" just to be safe

Reply:
Done.